data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -29,6 +29,7 @@
     private static final String columnsPropertyName = "Columns";
     private static final String columnNamePropertyName = "ColumnName";
     private static final String columnTypePropertyName = "ColumnType";
+    private static final String columnTypeSecondPropertyName = "ColumnType";
```

Comment:
This is the same as columnTypePropertyName

Reply:
Also, naming. I suggest columnTypeAlternateName

Reply:
fixed

Reply:
"DataType"